nav2_controller/doc/README.md

```diff
@@ -62,8 +75,7 @@ TaskStatus execute(path)
 ```
 ## Future Plans
 
-* Add support for updating the planned path before we've finished the current one.
-* Add support for recovery behaviors
-* Simplify parameters and remove obsolete parameters
-* Remove the direct inclusion of `costmap_2d` and replace it with calls to the world model node.
-* Port another planner to refine the interfaces to the planner and to the world model. There are not many local planners available for the navigation stack, it seems. However `Time Elastic Band` seems like an interesting possibility of the few that are out there, as it interprets the dynamic obstacles very differently.
+* Add support for recovery behaviors #49
```

Comment:
Do you want to link the issues here or just have the number?

Reply:
I thought github would magically turn them into links like it does in commit messages and PRs. I'll fix.